src/test/java/org/kiwiproject/dropwizard/poller/config/PollerHealthCheckConfigTest.java

```diff
@@ -51,6 +51,7 @@ void shouldSetDefaultValues_WhenExplicitlyGivenAllNulls(SoftAssertions softly) {
     private void assertDefaultValues(SoftAssertions softly) {
         // NOTE: Keeping SoftAssertions here because there will be more as I add more health checks
```

Comment:
This can be removed now right?

Reply:
No there is one more health check

Reply:
Yes, but you already have two soft asserts in here, thus it already makes sense that it is using `SoftAssertions`, right?

Reply:
Anyway, don't forget to remove it later...